parm/makefile

```diff
@@ -80,6 +81,7 @@ postxconfig-NT-hafs_ar_nosat.txt:    postcntrl_hafs_ar_nosat.xml    post_avblfld
 postxconfig-NT-hafs_ar_nosat_l65.txt: postcntrl_hafs_ar_nosat_l65.xml post_avblflds.xml
 postxconfig-NT-hafs.txt:             postcntrl_hafs.xml             post_avblflds.xml
 postxconfig-NT-hrrr.txt:             hrrr_postcntrl.xml             post_avblflds_raphrrr.xml
+postxconfig-NT-ifi.txt:              ifi_postcntrl.xml              post_avblflds_raphrrr.xml
```

Comment:
@SamuelTrahanNOAA I suggest switching to post_avblflds.xml instead of post_avblflds.xml.

Reply:
done